### DISCUSSION TITLE
다이어리 입/출력창에 대한 아이디어

### DISCUSSION THREAD

**Discussion by Leekyeongmi** [codestates/siphoon-note#77] [Ideas]
1. 텍스트 길이에 따른 유동적인 박스 크기 구현이 어렵다면, 어차피 `더보기`기능을 구현할 거라면, 오히려 상자 자체를 작게 만들고 한 번에 보여지는 상자의 개수 늘린다. 

2. **랜딩페이지에 보여지는 5가지 색감을 바탕색으로 추출해와서, 사용자가 배경화면 색이나 가능하면 이미지도 바꿀 수 있도록** 한다. (랜딩페이지의 다섯가지 색감이 충분히 블록같은 느낌을 줄 수 있을 것 같다.)

3. **카드가 추가되는 애니메이션**을 게임처럼 구현한다. (블록을 쌓듯)

4. 글쓰기 습관 형성에 도움을 주는 서비스니까, **placeholder 부분에 글감을 제안**해주면 어떨까? 

5. 추가) 내가 쓰려고 하는 글의 제목을 작성한다. 글의 상단에 n번째 일기 대신에, 내가 작성한 글의 글감이 무엇인지 보여준다.

![image](https://user-images.githubusercontent.com/86139013/159816725-35d20772-ba47-4aa6-8a07-2861faa9c8a6.png)

**Comment by gomarag** [comment]
제안하신 아이디어 다 좋네요! 
테마가 커스터마이징 되니까 더 프라이빗한 공간 같은 느낌도 들고요
그럼 백엔드에서는 글쓰기 키워드가 될 만한 단어를 db에 담고 있다가 랜덤하게 보내주거나 하면 좋을 것 같아요

**Reply by Leekyeongmi** [reply]
오! 네네 ㅎㅎ 아, 화면에서는 이모지가 사라졌는데.. 선택한 이모지는 다시 조합해서 테스트해볼 예정입니다!
